samples/Playground/Playground.Windows/ThemeSwitchPage.xaml

```diff
@@ -0,0 +1,39 @@
+﻿<Page
+    x:Class="Playground.ThemeSwitchPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
Add ThemeSwitchPage.xaml and ThemeSwitchPage.xaml.cs to the Shared project (where HomePage is)